### PR TITLE
AT_01.05_014 | Header>Visiting User Configure Page and Changing 

### PR DESCRIPTION
**Pull request by KseniyaParfianovich**
https://trello.com/c/XQeXVoar/1265-at0105014-headervisiting-user-configure-page-and-changing-user-information

### CODE REVIEW

**Review by matvlesia — APPROVED** [review]
Please remove only

File: cypress/e2e/headerUserConfigure.cy.js [hunk 1]
```diff
@@ -131,4 +131,18 @@ describe('Header User configure', () => {
         cy.get('#description').should('include.text', userDescription.textDescription);
     })
 
-});
\ No newline at end of file
+    it.only ('AT_01.05_014 | Header>Visiting User Configure Page and Changing User Information', () => {
+        cy.get("a[href^='/user/']>.jenkins-menu-dropdown-chevron")
+          .click({force: true});
```
Comment (Churs82) [inline cypress/e2e/headerUserConfigure.cy.js:136]: try to use realHover().click() instead of click({force:true}) next time

**Comment by KseniyaParfianovich** [comment]
> Please remove only

Thanks! I've changed it.

File: cypress/e2e/headerUserConfigure.cy.js (hunk 1 above)
Comment (KseniyaParfianovich) [reply cypress/e2e/headerUserConfigure.cy.js]: > try to use realHover().click() instead of click({force:true}) next time

Thanks! I've changed it.

**Pull request merged**